input/exercises/solutions/test_basic_1.md

```diff
@@ -0,0 +1,56 @@
+title: Test základních znalostí 1
+---
+>Tento dokument pracuje se dvěma obrázky: 
```

Comment:
Tohle by spíš mělo být v zadání... taky nahraď obrázek kytky **in**  třeba: "Obrázek kytky v proměnné A"